doc/version_history.rst

```diff
@@ -16,6 +16,8 @@ v2.0.0
 * Remove all simulator code from the CSC and connect to the simulator via TCP/IP.
 * Make ATPneumaticsCsc a Configurable CSC.
 * Add support for sending start, disable, enable and standby commands.
+* Add support for summary state and detailed state events and for FAULT state.
```

Comment:
Same comment for the final "and" word as another PR.

Reply:
I added support for two things:
- the events for summary state and detailed state
- the FAULT state